how-to/containerd-kata.md

```diff
@@ -77,8 +77,8 @@ Follow the instructions to [install Kata Containers](https://github.com/kata-con
 ### Install containerd with cri plugin
 
 > **Note:** `cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default. 
-> You do not need to install `cri` if you have containerd 1.1 or above. Just check if the `cri` plugin has been disabled 
-> in the containerd configuration file.
+> You do not need to install `cri` if you have containerd 1.1 or above. Just remove the `cri` plugin from the list of
+> `disabled_plugins` in the containerd configuration file (/etc/containerd/config.toml).
```

Comment:
Nit: `/etc/containerd/config.toml`, to conform to https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#files-and-command-names.